pontoon/teams/static/js/info.js

```diff
@@ -0,0 +1,48 @@
+/* global $ */
+$(function() {
+  var container = $('#main .container');
+
+  container.on('click', '.edit-info', function(e) {
+    e.preventDefault();
+    var infoBox = container.find('.info');
+    var content = infoBox.html();
+    var textArea = $('.read-write-info textarea').val($.trim(content));
+    $('.edit-info').hide();
+    $('.read-only-info').hide();
+    $('.read-write-info').toggleClass('hidden');
+    textArea.focus();
+  });
+
+  container.on('click', '.cancel', function(e) {
+    e.preventDefault();
+    $('.edit-info').show();
+    $('.read-only-info').show();
+    $('.read-write-info').toggleClass('hidden');
+    return false;
+  });
+
+  container.on('click', '.save', function(e) {
+    var textArea = $('.read-write-info textarea');
+    var content = textArea.val().replace(/(?:\r\n|\r|\n)/g, '<br>');
```

Comment:
Why not just remove those newline characters if they're at the end of the string?